services/clusters-keeper/src/simcore_service_clusters_keeper/data/prometheus.yml

```diff
@@ -0,0 +1,51 @@
+global:
+  scrape_interval: "30s"
+scrape_configs:
+  # Make Prometheus scrape itself for metrics.
+  - job_name: 'prometheus'
+    static_configs:
+    - targets: ['localhost:9090']
+  # Create a job for Docker Swarm containers.
+  - job_name: 'docker nodes'
+    dockerswarm_sd_configs:
+      - host: unix:///var/run/docker.sock
+        role: nodes
+    relabel_configs:
+      # Fetch metrics on port 9323.
+      - source_labels: [__meta_dockerswarm_node_address]
+        target_label: __address__
+        replacement: $1:9323
+      # Set hostname as instance label
+      - source_labels: [__meta_dockerswarm_node_hostname]
+        target_label: instance
+  # Create a job for Docker Swarm containers.
+  - job_name: 'docker tasks'
+    dockerswarm_sd_configs:
+      - host: unix:///var/run/docker.sock
+        role: tasks
+    relabel_configs:
+      # Only keep containers that should be running.
+      - source_labels: [__meta_dockerswarm_task_desired_state]
+        regex: running
+        action: keep
+      # Only keep tasks with a `prometheus_port` label.
+      - source_labels: [__meta_dockerswarm_service_label_prometheus_port]
+        regex: .+
+        action: keep
+      # Only keep containers that have a `prometheus-job` label.
+      - source_labels: [__meta_dockerswarm_service_label_prometheus_job]
+        regex: .+
+        action: keep
+      # Use the prometheus-job Swarm label as Prometheus job label.
+      - source_labels: [__meta_dockerswarm_service_label_prometheus_job]
+        target_label: job
+      # Specify the metric path if needed (optional)
+      - source_labels: [__meta_dockerswarm_service_label_prometheus_path]
+        target_label: __metrics_path__
+        regex: (.+)
+      # Use the `prometheus_port` Swarm label to set the __address__ for scraping.
```

Comment:
Is this a cleaner version of what we have in ops?

```
# Keep the containers IP and Port, very necessary for our setup DONT MESS WITH THIS
      - source_labels:
          [__address__, __meta_dockerswarm_service_label_prometheus_port]
        separator: ";"
        regex: "(.*):.*;(.*)"
        target_label: __address__
        replacement: $1:$2
```

your regex seems better

Reply:
did not even see we had it already. actually chatty proposed it, and I tried it;)